### DISCUSSION TITLE
Enable chart metadata edit in menu.

### DISCUSSION THREAD

**Discussion by akashkj** [apache/superset#19590] [Q&A / Help]
## Screenshot

![image](https://user-images.githubusercontent.com/14318173/162175205-5d09930f-4df5-453b-83f8-24c7da5eef12.png)

## Description
How do I get the highlighted option in chart menu, which was available in older versions.

I am able to access it directly with the url: /chart/edit/<slice_id>

**Comment by srinify** [comment]
I could be wrong but I believe this functionality was removed entirely from Superset. Moving to Discussions so Issues can be focused on bugs!

**Comment by akashkj** [comment]
Ok, since url still can be accessed directly, my assumption was that it's display is hidden under some permission.
